subprojects/core/src/main/java/org/gradle/caching/configuration/BuildCache.java

```diff
@@ -44,4 +47,17 @@
      * Sets whether pushing to the build cache is enabled.
      */
     void setPush(boolean enabled);
+
+    /**
+    * Provides a human friendly description of the type of this cache.
+    * e.g. Directory, HTTP
+    */
+    @Nullable
+    String getDisplayName();
```

Comment:
`type` has been chosen over `displayName` so `Describable` is not applicable anymore.